action.yml

```diff
@@ -21,8 +21,17 @@ inputs:
   previews:
     description: A comma-separated list of API previews to accept
   result-encoding:
-    description: Either "string" or "json" (default "json")—how the result will be encoded 
+    description: Either "string" or "json" (default "json")—how the result will be encoded
     default: json
+  retries:
+    description: "The number of times to retry a request"
+    default: "0"
+  retry-after:
+    description: "The number of seconds between retries. No effect unless `retries` is set."
```

Comment:
Would it make sense to default to exponential backoff rather than a fixed retry after?

Reply:
The plugin does actually do exponential back off [source](https://github.com/octokit/plugin-retry.js/blob/9a2443746c350b3beedec35cf26e197ea318a261/src/error-request.ts#L13). I just didn't call it out to keep it succinct

Reply:
actually looking deeper. I don't think the plugin is properly using the `retryAfter` parameter at all 🤦 . The interval is backed off an is always `(n+1)^2`. I think I may just remove this option.